showcase/app/templates/components/advanced-table.hbs

```diff
@@ -143,6 +181,94 @@
     </Hds::AdvancedTable>
   </div>
 
+  <Shw::Text::H2>Sticky column</Shw::Text::H2>
+
+  <div class="shw-component-advanced-table-fixed-dimensions-wrapper">
+    <Hds::AdvancedTable
+      @isSelectable={{true}}
+      @onSelectionChange={{this.onMultiSelectUsersSelectionChange__demo3}}
+      @model={{this.model.userData}}
+      @columns={{array
+        (hash key="id" label="ID" width="auto")
+        (hash key="name" label="Name")
+        (hash key="email" label="Email")
+        (hash key="role" label="Role")
+      }}
+      @hasStickyFirstColumn={{true}}
+      @isStriped={{true}}
+    >
+      <:body as |B|>
+        <B.Tr
+          @selectionKey={{B.data.id}}
+          @isSelected={{B.data.isSelected}}
+          @selectionAriaLabelSuffix="row #{{B.data.id}}"
+        >
+          <B.Th>{{B.data.id}}</B.Th>
+          <B.Td>{{B.data.name}}</B.Td>
+          <B.Td>{{B.data.email}}</B.Td>
+          <B.Td>{{B.data.role}}</B.Td>
+        </B.Tr>
+      </:body>
+    </Hds::AdvancedTable>
+  </div>
+
+  <Shw::Text::H3>Sticky column not selectable</Shw::Text::H3>
+
+  <div class="shw-component-advanced-table-fixed-dimensions-wrapper">
+    <Hds::AdvancedTable
+      @model={{this.model.userData}}
+      @columns={{array
+        (hash key="id" label="ID" width="auto")
+        (hash key="name" label="Name")
+        (hash key="email" label="Email")
+        (hash key="role" label="Role")
+      }}
+      @hasStickyFirstColumn={{true}}
+      @isStriped={{true}}
+    >
+      <:body as |B|>
+        <B.Tr>
+          <B.Th>{{B.data.id}}</B.Th>
+          <B.Td>{{B.data.name}}</B.Td>
+          <B.Td>{{B.data.email}}</B.Td>
+          <B.Td>{{B.data.role}}</B.Td>
+        </B.Tr>
+      </:body>
+    </Hds::AdvancedTable>
+  </div>
+
+  <Shw::Text::H2>Sticky header and sticky column</Shw::Text::H2>
```

Comment:
<img width="174" alt="Screenshot 2025-03-24 at 1 50 51 PM" src="https://github.com/user-attachments/assets/e81fc93d-e684-4c98-89cd-06f84efc4e3a" />

There's ever so slightly some text bleeding through in this example

Reply:
I am not sure that's some thing that could be controlled, is it? Wouldn't it depend on how the user moved around?

Reply:
This was pretty tricky to resolve, but I _think_ it happens less now.